stable/clamav/templates/deployment.yaml

```diff
@@ -63,6 +64,7 @@ spec:
         - name: clamd-config-volume
```

Comment:
this fix the repeated "values:" when freshclamConfig but  if .Values.clamdConfig then "values" will be repeated too, please delete line 63